### DISCUSSION TITLE
How to turn exported material files (.mat,props) back to .uasset,uexp?

### DISCUSSION THREAD

**Discussion by ColourfulPhoenix** [gildor2/UEViewer#246] [Q&A] [answered]
Hello Gildor!
I'm a newbie to UE4,trying to make a mod for PAL7.
I wonder if this is possible?if so,how can I do it?
Many Thanks :)

**Comment by gildor2** [comment, marked as answer]
No way.

**Reply by ColourfulPhoenix** [reply]
lol I see,thanks!

**Comment by ColourfulPhoenix** [comment]
btw Gildor could you please take a look at this?
https://forum.xentax.com/viewtopic.php?f=16&t=24643
can those .uexp files under material folders be edited?
